Library/Homebrew/unpack_strategy/dmg.rb

```diff
@@ -87,8 +87,27 @@ def eject(verbose: false)
         return unless path.exist?
 
         if tries > 1
+          disk_info = system_command(
```

Comment:
```suggestion
          disk_info = system_command!(
```

This should always succeed, otherwise `eject` would likely also fail.

Reply:
Added into commit.